lib/src/kbasesearchengine/search/ObjectData.java

```diff
@@ -526,5 +548,16 @@ public Builder withKeyProperty(final String key, final String property) {
             keyProps.put(key, property);
             return this;
         }
+
+        /** Adds the highlight fields to the object.
+         * @param highlight the map of fields returned from elasticsearch.
+         * @return this builder.
+         */
+        public Builder withNullableHighlight(final Map<String, ArrayList> highlight) {
```

Comment:
Needs tests in ObjectDataTest

Reply:
I should only need to modify the existing tests to include the field right?

Reply:
yep, take a look at the current tests for how to test the additional field correctly